srcjs/input_binding_date.js

```diff
@@ -16,10 +16,17 @@ $.extend(dateInputBinding, {
   setValue: function(el, value) {
     var date = this._newDate(value);
     // If date is invalid, do nothing
+    // Note: R's NA is null here, and isNaN(null) is false
```

Comment:
I think the change in behavior to `this._newDate()` is OK, but I'd prefer logic like this:

```js
if (value === null) {
  $(el).find('input').val('').datepicker('update');
  return;
}

// original code here
```

It's not a big deal - it's just a little easier to mentally process.